### DISCUSSION TITLE
Welcome to awesome-green-software Discussions!

### DISCUSSION THREAD

**Discussion by seanmcilroy29** [Green-Software-Foundation/awesome-green-software#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GustavoFranchella** [comment]
Hi everybody. 

First of all, I still wish you a venturous genuine 2022 V.0 (i.e. not a 2020 V.2). I am a 65 years old IT veteran who expressed his interest in green software almost one year ago in this article: https://www.linkedin.com/pulse/moores-law-global-warming-gustavo-franchella.

I've been using (enjoying) WeDeex|Edge for three days now. May the creators provide clarification on the issue raised below?

The plug-in description states: "Get a real-time simple indicator about the CO2 emission level linked to electrical power generation in France (more countries to come". 

My first thought was: France is renowned for having a high share of nuclear-generated power, then the consumption rate shall vary in other countries depending on the generation matrix mix. Then I thought: How do they compute the ordinary cross-border consumption? 

I'd be glad to know, and indeed, congratulations on the plug-in, even not knowing the innings, it is superb! Best regards.
